### ISSUE TITLE
More than just a sidebar!

### ISSUE THREAD

**Issue by shaunandrews** [ItsJonQ/g2#15]
I’m anxious to see this UI working outside of the editor sidebar. How can we map these patterns to the top toolbar, the block toolbar, and even outside of the editor into wp-admin.

**Comment by ItsJonQ** [comment]
Yes! I'm glad you brought these topics up. The components are designed to enable seamless integration into varying apps, environments, and features* (*Assuming JS + React can run)

The top/block toolbar are interesting real world candidates. I'm excited to prototype these out. That being said, I understand there is **a lot** (I mean a lot) that goes into the Toolbar in Gutenberg. So the prototypes will be approaching it (almost) purely from a UI + interaction perspective.

If you have any ideas for potential WP Admin UI / implementations, please lemme know :D

**Comment by ItsJonQ** [comment]
Technically still a Sidebar... but it's on the left side now 😂 

https://github.com/ItsJonQ/g2/issues/20
